src/codeu/chat/client/messageSecurity.java

```diff
@@ -0,0 +1,72 @@
+import sun.misc.BASE64Decoder;
+import sun.misc.BASE64Encoder;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.spec.SecretKeySpec;
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.Key;
+import java.security.NoSuchAlgorithmException;
+
+public class messageSecurity {
```

Comment:
It's a good idea to document your classes to give an indication of what they do, their APIs, etc. This helps people looking at your code understand the responsibilities of the class.